docs/topics/development/debugging.rst

```diff
@@ -90,6 +88,10 @@ If clicked, it looks like:
 
 .. image:: /screenshots/django-debug-toolbar-expanded.png
 
+You must know that using the Django Debug Toolbar will slow the website quite a
+lot. You can mitigate this by deselecting the checkbox next to the ``SQL``
+panel.
```

Comment:
Ping @andymckay @mstriemer @kumar303 : I think this might make it behave much better/faster (stronger?), following some discussion we had a long time about about the performance of the DjDT.